Jenkinsfile

```diff
@@ -98,7 +98,7 @@ pipeline {
 
   post {
     success {
-      cleanupAndNotify(currentBuild.currentResult)
+      cleanupAndNotify(currentBuild.currentResult, "#development", "@secrets-provider-for-k8s-owners")
```

Comment:
why did you add a notification in case of success?

Reply:
I rebased off of master here. @eladkug ?

Reply:
I didnt change the success section. 
I updated unsuccessful section only.

Reply:
weird.. okay I will just remove that again then